projects/storefrontstyles/scss/components/myaccount/_cx-my-account-v2-consent-management-form.scss

```diff
@@ -0,0 +1,57 @@
+%cx-my-account-v2-consent-management-form {
+  .cx-be-bold {
+    @include type('6');
+  }
+
+  .name {
+    @include type('6');
+  }
+
+  .form-check .form-check-label {
+    display: block;
+  }
+
+  .description {
+    @include type('');
+  }
+
+  .consent-container {
+    display: flex;
+  }
+
+  .consent-details {
+    margin: 0 0 0 2.815rem;
```

Comment:
Substitute margin-left and margin-right for margin-inline-start and margin-inline-start. We need these for directionality to work.
padding-top: 0
padding-inline-end: 0
padding-bottom: 0
padding-inline-start: 2.815rem